win32/build/confutils.js

```diff
@@ -3322,6 +3349,14 @@ function toolset_setup_intrinsic_cflags()
 			ERROR("Can't enable intrinsics, --with-codegen-arch passed with an incompatible option. ")
 		}
 
+		if (TARGET_ARCH == 'arm64') {
+			/* arm64 supports neon */
+			configure_subst.Add("PHP_SIMD_SCALE", 'NEON');
+			/* all offically supported arm64 cpu supports crc32 (TODO: to be confirmed) */
```

Comment:
would it be "overkill" (even possible, no idea) to attempt compiling a sample assembly/C, catching LASTEXITCODE ... to assess the situation ?

Reply:
I just cannot find a real world arm64 cpu without neon, plus, it's hard to check this on cross-compiling (native toolset is not available at that time, it's still in beta now).

For crc32 there's other checks done in #7703

Reply:
makes sense, fingers crossed then :-)